overlord/assertstate/validation_set_tracking.go

```diff
@@ -104,8 +104,26 @@ func UpdateValidationSet(st *state.State, tr *ValidationSetTracking) {
 	st.Set("validation-sets", vsmap)
 }
 
+// isForgetAllowedByModelAssertion checks whether a validation-set is controlled by
+// the model assertion. If the validation-set is set to 'enforce', then it's not possible
+// to forget it.
+func isForgetAllowedByModelAssertion(st *state.State, accountID, name string) error {
```

Comment:
(nitpick^2) "isFoo()" feels more like a bool return, here we return an "err" so maybe "validateForgotAllowedByModelAssertion" or something along this line? Or `canForgetValidationSet()` or somesuch.

But super minor nitpick to be clear.